spec/models/item_spec.rb

```diff
@@ -7,6 +7,10 @@
     it { is_expected.to validate_presence_of(:price_in_cents) }
     it { is_expected.to validate_presence_of(:image_url) }
     it { is_expected.to validate_uniqueness_of(:title) }
+    it "is false by default" do
+      item = create(:item)
+      expect(item.retired).to be false
+    end
   end
```

Comment:
Lets add a test, to test for boolean values
expect(item.retired).to be_a(Boolean)